src/briefcase-install.rst

```diff
@@ -1,23 +1,32 @@
-.. spelling::
+Setting Up Briefcase
```

Comment:
This should be ODK Briefcase to match what is currently done with Collect/Aggregate